companies/costco.md

```diff
@@ -0,0 +1,10 @@
+---
+forbes_rank: 142
+name: Costco Wholesale
+slug: Costco
```

Comment:
```suggestion
slug: costco
```

Here's some documentation about slugs - https://support.squarespace.com/hc/en-us/articles/205814578-URL-slugs#toc-url-slug-formatting-tips

They should be all lowercase :)